test/common/src/main/kotlin/org/codice/compliance/verification/binding/PostVerifier.kt

```diff
@@ -13,21 +13,20 @@
  */
 package org.codice.compliance.verification.binding
 
-import org.apache.cxf.rs.security.saml.sso.SSOConstants.RELAY_STATE
 import org.apache.cxf.rs.security.saml.sso.SSOConstants.SIGNATURE
 import org.codice.compliance.SAMLComplianceException
 import org.codice.compliance.SAMLComplianceExceptionMessage.*
 import org.codice.compliance.children
+import org.codice.compliance.saml.plugin.IdpPostResponse
 import org.codice.compliance.utils.TestCommon.Companion.EXAMPLE_RELAY_STATE
-import org.w3c.dom.Node
 
-class PostVerifier(responseDom: Node, parsedResponse: Map<String, String>, givenRelayState: Boolean): BindingVerifier(responseDom, parsedResponse, givenRelayState) {
+class PostVerifier(val response: IdpPostResponse) : BindingVerifier() {
     /**
      * Verify the response for a post binding
      */
     override fun verifyBinding() {
-        verifySsoPost(responseDom)
-        verifyPostRelayState(parsedResponse[RELAY_STATE], givenRelayState)
+        verifySsoPost(response)
```

Comment:
✏️ No need to pass the `response` down to the functions; they would already have access to the final `response` field on the class. They should also both be private.